test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -24,23 +24,46 @@
 import torchvision
 import onnx
 
-
-def ort_test_with_input(ort_sess, input, output, rtol, atol):
-    input, _ = torch.jit._flatten(input)
-    output, _ = torch.jit._flatten(output)
-
-    def to_numpy(tensor):
-        if tensor.requires_grad:
-            return tensor.detach().cpu().numpy()
-        else:
-            return tensor.cpu().numpy()
-
+def to_numpy(tensor):
+    if tensor.requires_grad:
+        return tensor.detach().cpu().numpy()
+    else:
+        return tensor.cpu().numpy()
+
+def convert_to_onnx(model, input=None, opset_version=9, example_outputs=None,
+                    do_constant_folding=True, keep_initializers_as_inputs=True, 
+                    dynamic_axes=None, input_names=None, output_names=None,
+                    fixed_batch_size=False, training=None):
+    # export the model to ONNX
+    f = io.BytesIO()
+    input_copy = copy.deepcopy(input)
+    torch.onnx._export(model, input_copy, f,
+                       opset_version=opset_version,
+                       example_outputs=example_outputs,
+                       do_constant_folding=do_constant_folding,
+                       keep_initializers_as_inputs=keep_initializers_as_inputs,
+                       dynamic_axes=dynamic_axes,
+                       input_names=input_names, output_names=output_names,
+                       fixed_batch_size=fixed_batch_size, training=training)
+
+    # compute onnxruntime output prediction
+    ort_sess = onnxruntime.InferenceSession(f.getvalue())
```

Comment:
Would it be possible to move ort session creation from this method to run_ort method?
It's just using the onnx file ('f').

Reply:
actually it's better to return ort_sess because of this line: https://github.com/pytorch/pytorch/blob/c446f0747a9988e7dcc215d1f8135e43e2c0b23b/test/onnx/test_pytorch_onnx_onnxruntime.py#L185